testfiles/1ZoneUncontrolledFourAlgorithms.idf

```diff
@@ -502,13 +502,17 @@
     0.7500000;               !- Visible Absorptance
 
   MaterialProperty:MoisturePenetrationDepth:Settings,
-    GP01,                    !- Name
-    0.004,                   !- Moisture Penetration Depth {m}
-    0.072549,                !- Moisture Equation Coefficient a {dimensionless}
-    0.397173,                !- Moisture Equation Coefficient b {dimensionless}
-    0.007774,                !- Moisture Equation Coefficient c {dimensionless}
-    11.7057;                 !- Moisture Equation Coefficient d {dimensionless}
-
+    GP01,                   !- Name
+    27.3004417577,                   !- Water Vapor Diffusion Resistance Factor {dimensionless}
+    0.072549,                   !- Moisture Equation Coefficient a {dimensionless}
+    0.397173,                   !- Moisture Equation Coefficient b {dimensionless}
+    0.007774,                   !- Moisture Equation Coefficient c {dimensionless}
+    11.7057,                   !- Moisture Equation Coefficient d {dimensionless}
+    0.004,                   !- Surface-layer penetration depth {m}
```

Comment:
Be sure to update field names in test files to match changes in the IDD, just for consistency.